apps/web/components/molecules/tabs/tabs.tsx

```diff
@@ -0,0 +1,53 @@
+import { Tab } from '@headlessui/react';
+import { Opinions } from 'molecules/opinions/opinions';
+import clsx from 'clsx';
+import type { ReactNode } from 'react';
+
+type TabObject = {
+	label: string;
+	content: ReactNode;
+};
+
+export const Tabs = () => {
+	const tabs: TabObject[] = [
+		{
+			label: 'Opinie',
+			content: <Opinions />,
+		},
+		{
+			label: 'Rozwiązania',
+			content: 'Tab 2',
+		},
+		{
+			label: 'Komentarze',
+			content: 'Komentarze',
+		},
+	];
```

Comment:
This doesn't need to be inside component function body, it doesn't depend on components lifecycle

Reply:
Where I should insert this?

Reply:
It **could** be moved to a different file or above function declaration, but I'm not really sure if it needs to be changed, you can leave it like that if that works for you.